packages/orbit-design-tokens/src/dictionary/utils/__tests__/is.test.ts

```diff
@@ -1,9 +1,9 @@
-import { isColor, isSpacing, isInternal, isNotInternal } from "../is";
+import { isColor, isSpacing, isInternal, isNotInternal, isBorderRadius } from "../is";
 
-const typeColor = {
-  type: "color",
+const tokenPlaceholder = ({ type = "color", internal = true }) => ({
```

Comment:
It looks like you don't need these default values, and in tests it's better to be explicit about what you're testing.
```suggestion
const tokenPlaceholder = ({ type, internal }) => ({
```